quickstarts/Tuning.ipynb

```diff
@@ -48,7 +48,7 @@
       "source": [
```

Comment:
<blockquote>a Google Sheets file</blockquote><p>IIRC this is only supported for a <em>public</em> Google Sheets URL.</p><p> </p><blockquote>The tuning API will read a csv file with pd.read_csv and handle it as a Mapping.&nbsp;</blockquote><p><code>pd.read_csv</code>  is an implementation detail, and I realise we have documented it, but we should limit any further leakage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>